src/hooks/bookmarks/use-all-bookmarks.ts

```diff
@@ -42,7 +41,7 @@ const useAllBookmarks = ({
 		['bookmarks'],
 		() => getAllBookmarks({ accessToken }),
 		{
-			enabled: enabled && !!accessToken,
+			enabled: enabled && isAuthenticated && !!accessToken,
```

Comment:
Instead of relying on the presence of the accessToken to check whether we are in a good state to make an API call, we should rely on something like `isAuthenticated`, which could be adjusted if we are in an error state where an accessToken may exist but it may not be valid.

Reply:
I suppose I could also use error or status here instead of storing something in state 🤔